app/controllers/search_results_controller.rb

```diff
@@ -0,0 +1,49 @@
+class SearchResultsController < ApplicationController
+  before_action :set_search_result, only: %i[show edit update destroy]
+
+  # GET /search_results or /search_results.json
```

Comment:
GET /search_results.json is not an available route so it should not be listed here.